docs/calculated-questions.md

```diff
@@ -0,0 +1,50 @@
+# Calculated Questions
+
+Caluma supports the concept of calculated questions, currently only of type float.
+This means we can create a question containing a JEXL expression (`calcExpression`), for which the answers are automatically generated from said expression.
+
+Example:
+We want to calculate the total costs of an order, which is dependent on a couple of other answers.
+For this we have a question `quantity` which is an integer, as well as `vat` and a `product-price`.
+
+The resulting `total-price` answer will be calculated from the JEXL expression:
+```
+'"quantity"|answer * ("product-price"|answer * (1 + "vat"|answer))'
+```
+
+### Implementation
+
+This feature makes heavy use of Django signals. There are two "main loops" for this.
+
+
+#### `calcDependents`
```

Comment:
I would move this fragment to internal documentation. We're not exposing it, and the user won't need to know about it

Reply:
Which internal documentation do you mean? ;)

Reply:
Maybe just a docstring in the corresponding class :wink:

Reply:
It doesn't really fit anywhere because there are just an number of signal together in a file... But yeah it doesn't fit in the user docs either.